index.html

```diff
@@ -12,124 +12,229 @@
   <body>
     <div class="main">
       <div class="header">
-        <img
-          class="triphouse-logo"
-          src="./images/logo_vector.png"
-          alt="TripHouse"
-        />
-        <div class="menu">
-          <div class="stays">
-            <h3 class="header-text">Stays</h3>
-            <hr class="stays-line" />
+        <svg class="header__logo">
+          <use href="./images/sprite.svg#logo" />
+        </svg>
+        <div class="header__menu">
+          <div class="header__menu--stays">
+            <h3 class="header__text">Stays</h3>
```

Comment:
header__menu--stays
header__menu--text
 if you use BEM - be careful and keep the rules))